package/MDAnalysis/analysis/align.py

```diff
@@ -700,8 +723,6 @@ def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
     .. versionadded:: 0.10.0
 
     """
-    import Bio.pairwise2
-
     aln = Bio.pairwise2.align.globalms(
```

Comment:
Do you mean "covers" for "overs"?

> do we say in the module docs that MDAnalysis overs sequence alignment?

Not overtly, but the function is documented as a helper and it is necessary for semi-smart superposition. I wouldn't advertise it broadly because that's a side thing for `align`.